pkg/sdkserver/localsdk.go

```diff
@@ -498,7 +605,7 @@ func (l *LocalSDKServer) EqualSets(expected, received []string) bool {
 func (l *LocalSDKServer) compare() {
 	if l.testMode {
 		if !l.EqualSets(l.expectedSequence, l.requestSequence) {
-			l.logger.Errorf("Testing Failed %v %v", l.expectedSequence, l.requestSequence)
+			logrus.WithField("expected", l.expectedSequence).WithField("received", l.requestSequence).Info("Testing Failed")
```

Comment:
@aLekSer I snuck a small tweak to your conformance test reporting in here. WDYT?

Reply:
Great, now it is much more readable. But same as above:
```suggestion
			l.logger.WithField("expected", l.expectedSequence).WithField("received", l.requestSequence).Info("Testing Failed")
```